docker/docker-compose-run.yml

```diff
@@ -19,7 +19,7 @@ services:
         source: ./
         target: /code/
   relayer:
-    image: avaplatform/awm-relayer:v0.2.2
+    image: avaplatform/awm-relayer:v0.2.2-beta
```

Comment:
Let's pin this to an actual release

Reply:
will update for a new release when https://github.com/ava-labs/awm-relayer/pull/50 is merged